extension/runtime/src/main/java/io/quarkiverse/temporal/WorkerFactoryRecorder.java

```diff
@@ -78,12 +86,23 @@ public void createWorker(String name, List<Class<?>> workflows,
         for (var activity : activities) {
             worker.registerActivitiesImplementations(CDI.current().select(activity).get());
         }
-
     }
 
     public void startWorkerFactory(ShutdownContext shutdownContext) {
         WorkerFactory workerFactory = CDI.current().select(WorkerFactory.class).get();
         workerFactory.start();
+
+        // bridge Temporal OpenTracing to OpenTelemetry
+        if (buildtimeConfig.enableTelemetry()) {
+            OpenTelemetry openTelemetry = CDI.current().select(OpenTelemetry.class).get();
+            if (openTelemetry == null) {
+                throw new IllegalStateException("OpenTelemetry not available");
+            }
+
+            io.opentracing.Tracer tracer = OpenTracingShim.createTracerShim(openTelemetry);
```

Comment:
how is the tracer added to the interceptor ?

Reply:
is it relying on a global singleton somewhere ?

Reply:
in this readme: https://github.com/temporalio/sdk-java/tree/master/temporal-opentracing

I see that the shim is added to the global context like this:

```
    io.opentracing.Tracer tracer = OpenTracingShim.createTracerShim(openTelemetry);
    GlobalTracer.registerIfAbsent(tracer);
```

Reply:
have you been able to confirm that it is actually working ?

Reply:
Yes when you even click on the Dev Ui car workers i can. See temporal connect to OpenTelemetry.